camelConnector/README.md

```diff
@@ -0,0 +1,276 @@
+in the Vantiq system. For an example of the definition, 
+please see the [*camelConnectorImpl.json*](src/test/resources/camelConnectorImpl.json) file
+located in the *src/test/resources* directory.
+
+# Setting Up Your Machine <a name="machine" id="machine"></a>
+
+## Repository Contents
+
+The _connector_ directory ('src/main/java/io/vantiq/extsrc/camelconn/connector`) contains the following files:
+
+*   **CamelMain** -- The main function for the program. Connects to sources as specified in a
+    configuration file.
+*   **CamelCore** -- Coordinates the connections to the database, and sends the resulting data back to VANTIQ Modelo if
+    necessary.
+*   **CamelHandleConfiguration** -- Sets up the Apache Camel routes based on the source's configuration document, and
+    initializes the queryHandler and publishHandler.
+
+The _discovery_ directory ('src/main/java/io/vantiq/extsrc/camelconnec/discover`) contains a variety of classes
+implementing the discovery, resolution, and provisioning of the components necessary to run the Apache Camel
+application configured as part of the source.
+
+## How to Run the Program <a name="runtheconnector" id="runtheconnector"></a>
+
+1.  Clone this repository (vantiq-extension-sources) and navigate into `<repo location>/vantiq-extension-sources`.
+2.  Run `./gradlew camelConnector:assemble`.
+3.  Navigate to `<repo location>/vantiq-extension-sources/camelConnector/build/distributions`. The zip and tar files both contain 
+    the same files, so choose whichever you prefer.
+4.  Uncompress the file in the location that you would like to install the program.
+5.  Run `<install location>/camelConnector/bin/camelConnector` with a local server.config file or specifying the [server config file](#serverConfig) as the first argument. Note that the `server.config` file can be placed in the `<install location>/camelConnector/serverConfig/server.config` or `<install location>/camelConnector/server.config` locations.
+
+### Changing Camel Versions
+
+The camelConnector has been tested with the version of Apache Camel defined in the `build.gradle` file.
+If you wish to change this, you will need to perform the following actions:
+
+1. Change the `camelVersion` property in the `build.gradle` file.
+2. Update the `artifactMap.json` found in `src/main/resources`. This file is the list of components and data formats provided by the Camel version in question.  To update this map, you will perform the following actions.
+   1. Clone the [Apache Camel source repository](https://github.com/apache/camel)
+   2. Set the `camelRoot` gradle property to the the root directory of the cloned repo above.
+   3. Run the `./gradlew camelConnector:generateComponentList` command.  This will take a little while as it examines the source code in question to determine the set of compoenents and data formats provided. This list is used when determining the set of libraries required to provision the connector for your Apache Camel application.
```

Comment:
```suggestion
   3. Run the `./gradlew camelConnector:generateComponentList` command.  This will take a little while as it examines the source code in question to determine the set of components and data formats provided. This list is used when determining the set of libraries required to provision the connector for your Apache Camel application.
```